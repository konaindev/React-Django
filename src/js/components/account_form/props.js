export const props = {
  steps: [
    { name: "Set Password", isComplete: true },
    { name: "Complete Account", isActive: true }
  ],
  title: "Complete your account",
  subtitle:
    "We need just a bit more information about you to complete your account."
};
