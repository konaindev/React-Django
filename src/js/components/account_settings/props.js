const user = {
  account_id: 1,
  account_name: "Remarkably",
  account_settings_url: "/users/account-settings",
  email: "test@psl.com",
  logout_url: "/users/logout/",
  user_id: "usr_jjzpeyfeshzpaha5"
};

const itemsOrder = ["profile", "lock", "email"];

const rules = [
  {
    label: "Be at least 8 characters",
    key: "length"
  },
  {
    label: "Contain alphabetic characters",
    key: "characters"
  },
  {
    label: "Not match personal information",
    key: "personal"
  },
  {
    label: "Not be a commonly used password",
    key: "used"
  }
];

const company_roles = [
  { label: "Owner", value: "owner" },
  { label: "Developer", value: "developer" },
  { label: "Asset Manager", value: "asset_manager" },
  { label: "Property Manager", value: "property_manager" }
];

const office_options = [
  { label: "Global", value: 1 },
  { label: "National", value: 2 },
  { label: "Regional", value: 3 },
  { label: "Other", value: 4 }
];

const office_countries = [
  { label: "United States of America", value: "USA" },
  { label: "United Kingdom", value: "GBR" }
];

const us_state_list = [
  { label: "Washington", value: "Washington" },
  { label: "New York", value: "New York" }
];

const gb_county_list = [
  { label: "London", value: "London" },
  { label: "Aberdeen", value: "Aberdeen" }
];

const profile = {
  avatar_url:
    "https://lh3.googleusercontent.com/-cQLcFi7r2uc/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rfoCSVbR8qVruV55uAYdSC-znVn2w.CMID/s96-c/photo.jpg",
  first_name: "Phillip",
  last_name: "McPhillipson",
  title: "Founder",
  phone_country_code: "",
  phone: "",
  phone_ext: "",
  company: { label: "Glacier Associates", value: "123456789" },
  company_roles: ["owner", "asset_manager", "property_manager"],
  office_street: "1730 Minor Avenue",
  office_city: "Lansing",
  office_state: { label: "Michigan", value: "Michigan" },
  office_zip: "11111",
  office_country: { label: "United States of America", value: "USA" },
  office_address: "1730 Minor Avenue, Lansing, MI",
  office_name: "Michigan",
  office_type: 3
};

const tabsOrder = ["portfolio", "group", "property"];

const initialTab = "portfolio";

export const props = {
  rules,
  profile,
  user,
  company_roles,
  office_options,
  itemsOrder,
  tabsOrder,
  initialTab,
  office_countries,
  us_state_list,
  gb_county_list,
  dispatch: () => {}
};
