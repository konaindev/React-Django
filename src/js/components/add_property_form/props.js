export const props = {
  post_url: "/projects/new",
  states: [
    { id: "AL", name: "AL" },
    { id: "GA", name: "GA" },
    { id: "NE", name: "NE" },
    { id: "OR", name: "OR" },
    { id: "WY", name: "WY" }
  ],
  packages: [
    { id: "accelerate", name: "Accelerate" },
    { id: "optimize", name: "Optimize" },
    { id: "ground", name: "Ground Up" },
    { id: "other", name: "Not Sure" }
  ]
};
