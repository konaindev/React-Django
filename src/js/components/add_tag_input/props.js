const suggestedTags = [
  { name: "Tag 1", count: 5 },
  { name: "Test Tag 2", count: 3 },
  { name: "Lake Highland Properties 3", count: 2 },
  { name: "New Tag", count: 1 }
];

export default { suggestedTags };
