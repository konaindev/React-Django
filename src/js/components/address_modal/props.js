export default {
  isOpen: true,
  addresses: {
    suggested_address: {
      office_city: "Washington",
      office_state: "DC",
      office_street: "1600 Pennsylvania Ave NW",
      office_zip: "20500",
      office_country: "United States"
    }
  },
  dispatch_type: "FAKE DISPATCH"
};
