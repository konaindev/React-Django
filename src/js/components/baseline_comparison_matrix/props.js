import {
  baselineReport,
  baselineReportWithoutCompetitors,
  baselineReportWithOneCompetitor
} from "../common_report/props";

export const props = { report: baselineReport };
export const no_competitor_props = { report: baselineReportWithoutCompetitors };
export const one_competitor_props = {
  report: baselineReportWithOneCompetitor
};
