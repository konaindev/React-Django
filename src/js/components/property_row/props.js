import { props as usersProps } from "../user_icon_list/props";

export const props = {
  property_id: "pro_ojso5xourpuai2vs",
  property_name: "Meridian SLC",
  address: "123 4th Avenue, Seattle WA 98106",
  image_url: "https://i.imgur.com/UEH4gfU.jpg",
  performance_rating: 2,
  report_url:
    "https://app.remarkably.io/projects/pro_nqcu73oiinomuvn7/baseline/",
  members: usersProps.users
};
