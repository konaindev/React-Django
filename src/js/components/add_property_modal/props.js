import { states } from "../add_property_form/states";

export const props = {
  formProps: {
    post_url: null,
    packages: [
      { id: "accelerate", name: "Accelerate" },
      { id: "optimize", name: "Optimize" },
      { id: "ground", name: "Ground Up" },
      { id: "other", name: "Not Sure" }
    ],
    states
  }
};
