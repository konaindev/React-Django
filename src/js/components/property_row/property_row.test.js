import renderer from "react-test-renderer";
import { MemoryRouter } from "react-router-dom";

import PropertyRow from "./index";
import { props } from "./props";

describe("PropertyRow", () => {
  it("renders on-track", () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <PropertyRow {...props} />
        </MemoryRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("renders off-track", () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <PropertyRow {...props} performance_rating={0} />
        </MemoryRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("renders at-risk", () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <PropertyRow {...props} performance_rating={1} />
        </MemoryRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("renders selection mode", () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <PropertyRow {...props} selection_mode={true} />
        </MemoryRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("renders selected", () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <PropertyRow {...props} selection_mode={true} selected={true} />
        </MemoryRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
