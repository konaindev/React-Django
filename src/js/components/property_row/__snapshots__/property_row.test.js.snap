// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PropertyRow renders at-risk 1`] = `
<div
  className="panel panel--default property-row"
>
  <div
    className="property-row__selector"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="property-row__image"
      style={
        Object {
          "backgroundImage": "url(\\"https://i.imgur.com/UEH4gfU.jpg\\")",
        }
      }
    />
    <div
      className="property-row__selector-hover"
    />
    <div
      className="property-row__tick"
    />
  </div>
  <div
    className="property-row__name"
  >
    Meridian SLC
  </div>
  <div
    className="property-row__address"
  >
    123 4th Avenue, Seattle WA 98106
  </div>
  <div
    className="property-row__link-container"
  >
    <a
      className="property-row__link"
      href="https://app.remarkably.io/projects/pro_nqcu73oiinomuvn7/baseline/"
    >
      View Property
    </a>
  </div>
  <div
    className="property-row__health"
  >
    <div
      className="property-status property-status--at-risk"
    >
      AT RISK
    </div>
  </div>
  <div
    className="property-row__members"
  >
    <div
      className="user-icon-list"
    >
      <div
        className="user-icon user-icon-list__icon"
        data-index={0}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "backgroundColor": "#006EFF",
            "zIndex": 4,
          }
        }
      >
        IK
        <div
          className="user-icon__avatar"
          style={
            Object {
              "backgroundImage": "url()",
            }
          }
        />
      </div>
      <div
        className="user-icon user-icon-list__icon"
        data-index={1}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "backgroundColor": "#6B29BE",
            "zIndex": 3,
          }
        }
      >
        PN
        <div
          className="user-icon__avatar"
          style={
            Object {
              "backgroundImage": "url(/avatar.jpg)",
            }
          }
        />
      </div>
      <div
        className="user-icon user-icon-list__icon"
        data-index={2}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "backgroundColor": "#006EFF",
            "zIndex": 2,
          }
        }
      >
        DC
        <div
          className="user-icon__avatar"
          style={
            Object {
              "backgroundImage": "url(https://robohash.org/architectoaccusantiumdolorem.jpg?size=50x50&set=set1&bgset=bg1)",
            }
          }
        />
      </div>
      <div
        className="user-icon user-icon-list__icon"
        data-index={3}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "backgroundColor": "#6B29BE",
            "zIndex": 1,
          }
        }
      >
        UA
        <div
          className="user-icon__avatar"
          style={
            Object {
              "backgroundImage": "url(https://robohash.org/estnecessitatibusporro.jpg?size=50x50&set=set1&bgset=bg1)",
            }
          }
        />
      </div>
      <div
        className="user-icon user-icon-list__icon"
        data-index={4}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "backgroundColor": "#006EFF",
            "zIndex": 0,
          }
        }
      >
        SC
        <div
          className="user-icon__avatar"
          style={
            Object {
              "backgroundImage": "url(https://robohash.org/esseconsequaturconsequatur.jpg?size=50x50&set=set1&bgset=bg1)",
            }
          }
        />
      </div>
      <div
        className="user-icon-list__count"
      >
        <span>
          +
          10
        </span>
      </div>
    </div>
  </div>
</div>
`;

exports[`PropertyRow renders off-track 1`] = `
<div
  className="panel panel--default property-row"
>
  <div
    className="property-row__selector"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="property-row__image"
      style={
        Object {
          "backgroundImage": "url(\\"https://i.imgur.com/UEH4gfU.jpg\\")",
        }
      }
    />
    <div
      className="property-row__selector-hover"
    />
    <div
      className="property-row__tick"
    />
  </div>
  <div
    className="property-row__name"
  >
    Meridian SLC
  </div>
  <div
    className="property-row__address"
  >
    123 4th Avenue, Seattle WA 98106
  </div>
  <div
    className="property-row__link-container"
  >
    <a
      className="property-row__link"
      href="https://app.remarkably.io/projects/pro_nqcu73oiinomuvn7/baseline/"
    >
      View Property
    </a>
  </div>
  <div
    className="property-row__health"
  >
    <div
      className="property-status property-status--off-track"
    >
      OFF TRACK
    </div>
  </div>
  <div
    className="property-row__members"
  >
    <div
      className="user-icon-list"
    >
      <div
        className="user-icon user-icon-list__icon"
        data-index={0}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "backgroundColor": "#006EFF",
            "zIndex": 4,
          }
        }
      >
        IK
        <div
          className="user-icon__avatar"
          style={
            Object {
              "backgroundImage": "url()",
            }
          }
        />
      </div>
      <div
        className="user-icon user-icon-list__icon"
        data-index={1}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "backgroundColor": "#6B29BE",
            "zIndex": 3,
          }
        }
      >
        PN
        <div
          className="user-icon__avatar"
          style={
            Object {
              "backgroundImage": "url(/avatar.jpg)",
            }
          }
        />
      </div>
      <div
        className="user-icon user-icon-list__icon"
        data-index={2}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "backgroundColor": "#006EFF",
            "zIndex": 2,
          }
        }
      >
        DC
        <div
          className="user-icon__avatar"
          style={
            Object {
              "backgroundImage": "url(https://robohash.org/architectoaccusantiumdolorem.jpg?size=50x50&set=set1&bgset=bg1)",
            }
          }
        />
      </div>
      <div
        className="user-icon user-icon-list__icon"
        data-index={3}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "backgroundColor": "#6B29BE",
            "zIndex": 1,
          }
        }
      >
        UA
        <div
          className="user-icon__avatar"
          style={
            Object {
              "backgroundImage": "url(https://robohash.org/estnecessitatibusporro.jpg?size=50x50&set=set1&bgset=bg1)",
            }
          }
        />
      </div>
      <div
        className="user-icon user-icon-list__icon"
        data-index={4}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "backgroundColor": "#006EFF",
            "zIndex": 0,
          }
        }
      >
        SC
        <div
          className="user-icon__avatar"
          style={
            Object {
              "backgroundImage": "url(https://robohash.org/esseconsequaturconsequatur.jpg?size=50x50&set=set1&bgset=bg1)",
            }
          }
        />
      </div>
      <div
        className="user-icon-list__count"
      >
        <span>
          +
          10
        </span>
      </div>
    </div>
  </div>
</div>
`;

exports[`PropertyRow renders on-track 1`] = `
<div
  className="panel panel--default property-row"
>
  <div
    className="property-row__selector"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="property-row__image"
      style={
        Object {
          "backgroundImage": "url(\\"https://i.imgur.com/UEH4gfU.jpg\\")",
        }
      }
    />
    <div
      className="property-row__selector-hover"
    />
    <div
      className="property-row__tick"
    />
  </div>
  <div
    className="property-row__name"
  >
    Meridian SLC
  </div>
  <div
    className="property-row__address"
  >
    123 4th Avenue, Seattle WA 98106
  </div>
  <div
    className="property-row__link-container"
  >
    <a
      className="property-row__link"
      href="https://app.remarkably.io/projects/pro_nqcu73oiinomuvn7/baseline/"
    >
      View Property
    </a>
  </div>
  <div
    className="property-row__health"
  >
    <div
      className="property-status property-status--on-track"
    >
      ON TRACK
    </div>
  </div>
  <div
    className="property-row__members"
  >
    <div
      className="user-icon-list"
    >
      <div
        className="user-icon user-icon-list__icon"
        data-index={0}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "backgroundColor": "#006EFF",
            "zIndex": 4,
          }
        }
      >
        IK
        <div
          className="user-icon__avatar"
          style={
            Object {
              "backgroundImage": "url()",
            }
          }
        />
      </div>
      <div
        className="user-icon user-icon-list__icon"
        data-index={1}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "backgroundColor": "#6B29BE",
            "zIndex": 3,
          }
        }
      >
        PN
        <div
          className="user-icon__avatar"
          style={
            Object {
              "backgroundImage": "url(/avatar.jpg)",
            }
          }
        />
      </div>
      <div
        className="user-icon user-icon-list__icon"
        data-index={2}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "backgroundColor": "#006EFF",
            "zIndex": 2,
          }
        }
      >
        DC
        <div
          className="user-icon__avatar"
          style={
            Object {
              "backgroundImage": "url(https://robohash.org/architectoaccusantiumdolorem.jpg?size=50x50&set=set1&bgset=bg1)",
            }
          }
        />
      </div>
      <div
        className="user-icon user-icon-list__icon"
        data-index={3}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "backgroundColor": "#6B29BE",
            "zIndex": 1,
          }
        }
      >
        UA
        <div
          className="user-icon__avatar"
          style={
            Object {
              "backgroundImage": "url(https://robohash.org/estnecessitatibusporro.jpg?size=50x50&set=set1&bgset=bg1)",
            }
          }
        />
      </div>
      <div
        className="user-icon user-icon-list__icon"
        data-index={4}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "backgroundColor": "#006EFF",
            "zIndex": 0,
          }
        }
      >
        SC
        <div
          className="user-icon__avatar"
          style={
            Object {
              "backgroundImage": "url(https://robohash.org/esseconsequaturconsequatur.jpg?size=50x50&set=set1&bgset=bg1)",
            }
          }
        />
      </div>
      <div
        className="user-icon-list__count"
      >
        <span>
          +
          10
        </span>
      </div>
    </div>
  </div>
</div>
`;

exports[`PropertyRow renders selected 1`] = `
<div
  className="panel panel--default property-row property-row--selection-mode property-row--selected"
>
  <div
    className="property-row__selector"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="property-row__image"
      style={
        Object {
          "backgroundImage": "url(\\"https://i.imgur.com/UEH4gfU.jpg\\")",
        }
      }
    />
    <div
      className="property-row__selector-hover"
    />
    <div
      className="property-row__tick"
    />
  </div>
  <div
    className="property-row__name"
  >
    Meridian SLC
  </div>
  <div
    className="property-row__address"
  >
    123 4th Avenue, Seattle WA 98106
  </div>
  <div
    className="property-row__link-container"
  >
    <a
      className="property-row__link"
      href="https://app.remarkably.io/projects/pro_nqcu73oiinomuvn7/baseline/"
    >
      View Property
    </a>
  </div>
  <div
    className="property-row__health"
  >
    <div
      className="property-status property-status--on-track"
    >
      ON TRACK
    </div>
  </div>
  <div
    className="property-row__members"
  >
    <div
      className="user-icon-list"
    >
      <div
        className="user-icon user-icon-list__icon"
        data-index={0}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "backgroundColor": "#006EFF",
            "zIndex": 4,
          }
        }
      >
        IK
        <div
          className="user-icon__avatar"
          style={
            Object {
              "backgroundImage": "url()",
            }
          }
        />
      </div>
      <div
        className="user-icon user-icon-list__icon"
        data-index={1}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "backgroundColor": "#6B29BE",
            "zIndex": 3,
          }
        }
      >
        PN
        <div
          className="user-icon__avatar"
          style={
            Object {
              "backgroundImage": "url(/avatar.jpg)",
            }
          }
        />
      </div>
      <div
        className="user-icon user-icon-list__icon"
        data-index={2}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "backgroundColor": "#006EFF",
            "zIndex": 2,
          }
        }
      >
        DC
        <div
          className="user-icon__avatar"
          style={
            Object {
              "backgroundImage": "url(https://robohash.org/architectoaccusantiumdolorem.jpg?size=50x50&set=set1&bgset=bg1)",
            }
          }
        />
      </div>
      <div
        className="user-icon user-icon-list__icon"
        data-index={3}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "backgroundColor": "#6B29BE",
            "zIndex": 1,
          }
        }
      >
        UA
        <div
          className="user-icon__avatar"
          style={
            Object {
              "backgroundImage": "url(https://robohash.org/estnecessitatibusporro.jpg?size=50x50&set=set1&bgset=bg1)",
            }
          }
        />
      </div>
      <div
        className="user-icon user-icon-list__icon"
        data-index={4}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "backgroundColor": "#006EFF",
            "zIndex": 0,
          }
        }
      >
        SC
        <div
          className="user-icon__avatar"
          style={
            Object {
              "backgroundImage": "url(https://robohash.org/esseconsequaturconsequatur.jpg?size=50x50&set=set1&bgset=bg1)",
            }
          }
        />
      </div>
      <div
        className="user-icon-list__count"
      >
        <span>
          +
          10
        </span>
      </div>
    </div>
  </div>
</div>
`;

exports[`PropertyRow renders selection mode 1`] = `
<div
  className="panel panel--default property-row property-row--selection-mode"
>
  <div
    className="property-row__selector"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="property-row__image"
      style={
        Object {
          "backgroundImage": "url(\\"https://i.imgur.com/UEH4gfU.jpg\\")",
        }
      }
    />
    <div
      className="property-row__selector-hover"
    />
    <div
      className="property-row__tick"
    />
  </div>
  <div
    className="property-row__name"
  >
    Meridian SLC
  </div>
  <div
    className="property-row__address"
  >
    123 4th Avenue, Seattle WA 98106
  </div>
  <div
    className="property-row__link-container"
  >
    <a
      className="property-row__link"
      href="https://app.remarkably.io/projects/pro_nqcu73oiinomuvn7/baseline/"
    >
      View Property
    </a>
  </div>
  <div
    className="property-row__health"
  >
    <div
      className="property-status property-status--on-track"
    >
      ON TRACK
    </div>
  </div>
  <div
    className="property-row__members"
  >
    <div
      className="user-icon-list"
    >
      <div
        className="user-icon user-icon-list__icon"
        data-index={0}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "backgroundColor": "#006EFF",
            "zIndex": 4,
          }
        }
      >
        IK
        <div
          className="user-icon__avatar"
          style={
            Object {
              "backgroundImage": "url()",
            }
          }
        />
      </div>
      <div
        className="user-icon user-icon-list__icon"
        data-index={1}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "backgroundColor": "#6B29BE",
            "zIndex": 3,
          }
        }
      >
        PN
        <div
          className="user-icon__avatar"
          style={
            Object {
              "backgroundImage": "url(/avatar.jpg)",
            }
          }
        />
      </div>
      <div
        className="user-icon user-icon-list__icon"
        data-index={2}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "backgroundColor": "#006EFF",
            "zIndex": 2,
          }
        }
      >
        DC
        <div
          className="user-icon__avatar"
          style={
            Object {
              "backgroundImage": "url(https://robohash.org/architectoaccusantiumdolorem.jpg?size=50x50&set=set1&bgset=bg1)",
            }
          }
        />
      </div>
      <div
        className="user-icon user-icon-list__icon"
        data-index={3}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "backgroundColor": "#6B29BE",
            "zIndex": 1,
          }
        }
      >
        UA
        <div
          className="user-icon__avatar"
          style={
            Object {
              "backgroundImage": "url(https://robohash.org/estnecessitatibusporro.jpg?size=50x50&set=set1&bgset=bg1)",
            }
          }
        />
      </div>
      <div
        className="user-icon user-icon-list__icon"
        data-index={4}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          Object {
            "backgroundColor": "#006EFF",
            "zIndex": 0,
          }
        }
      >
        SC
        <div
          className="user-icon__avatar"
          style={
            Object {
              "backgroundImage": "url(https://robohash.org/esseconsequaturconsequatur.jpg?size=50x50&set=set1&bgset=bg1)",
            }
          }
        />
      </div>
      <div
        className="user-icon-list__count"
      >
        <span>
          +
          10
        </span>
      </div>
    </div>
  </div>
</div>
`;
